tekton/predicates.go

```diff
@@ -39,8 +39,8 @@ func IntegrationOrBuildPipelineRunFinishedPredicate() predicate.Predicate {
 			return false
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			return (IsIntegrationPipelineRun(e.ObjectNew) || IsBuildPipelineRun(e.ObjectNew)) &&
-				hasPipelineRunStateChangedToFinished(e.ObjectOld, e.ObjectNew)
+			return (IsBuildPipelineRun(e.ObjectNew)) &&
+				hasPipelineRunBeenChangedToSigned(e.ObjectOld, e.ObjectNew) // this signes only finished pipelines
```

Comment:
Would this capture the PLRs which have finished?
The methods `IsBuildPipelineRun()` and `hasPipelineRunBeenChangedToSigned()` don't check if the PLR is finished, what am I missing?

Reply:
maybe I should rephrase the comment, while I'm reading it now, looks little bit braindead

Reply:
no im braindead, the comment makes sense now but can also be changed to something like `// only finished pipelines are signed` because im not sure what "this" refers to in the comment.

Reply:
updated